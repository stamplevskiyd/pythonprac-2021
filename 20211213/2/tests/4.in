[44, 88, 24, 69, 64, 28, 91, 0, 47, 73, 37, 7, 80, 73, 69, 43, 77, 23, 65, 71, 0, 43, 70, 2, 53, 6, 98, 68, 98, 10, 23, 19, 68, 90, 92, 15, 14, 1, 56, 59, 46, 48, 34, 11, 7, 87, 9, 85]