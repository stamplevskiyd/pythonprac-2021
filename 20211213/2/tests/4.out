[0, 0, 1, 2, 6, 7, 7, 9, 10, 11, 14, 15, 19, 23, 23, 24, 28, 34, 37, 43, 43, 44, 46, 47, 48, 53, 56, 59, 64, 65, 68, 68, 69, 69, 70, 71, 73, 73, 77, 80, 85, 87, 88, 90, 91, 92, 98, 98]
